### ISSUE TITLE
disabled dropdown tooltip

### ISSUE THREAD

**Issue by jschuler** [patternfly/patternfly#3680]
Similar to having the pf-m-aria-disabled modifier for buttons to visually make it looked disabled but still functional, can it be done for disabled dropdown/menu items as well?
This came up here recently
https://github.com/patternfly/patternfly-react/issues/5127

**Comment by mcoker** [comment]
I'm in favor of supporting this - @mcarrano @jessiehuff thoughts? Did this come up with the previous `pf-m-aria-disabled` work?

**Comment by mcarrano** [comment]
I think it's definitely something to consider @mcoker . I'm also copying in @andybraren as he took on a design issue to consider how we should support this.

**Comment by jessiehuff** [comment]
I think this would make sense to support. We started going down that road with [button](https://www.patternfly.org/v4/components/button/#aria-disabled-button-with-tooltip) like Joachim mentions above. We continued with it for dropdown in JAWS (allowing [disabled dropdown items to receive focus](https://github.com/patternfly/patternfly-react/pull/5044) which enabled JAWS users to see and announce them) which likely allows keyboard users to see tooltips on disabled items. So I think it would make sense to also allow mouse users to see these tooltips.
